vnext/Microsoft.ReactNative.Cxx/JSValue.h

```diff
@@ -51,7 +51,7 @@ struct JSValueObject : std::map<std::string, JSValue, std::less<>> {
   JSValueObject(JSValueObject const &) = delete;
 
   // Default move constructor.
-  JSValueObject(JSValueObject &&) = default;
+  JSValueObject(JSValueObject &&) noexcept = default;
```

Comment:
>noexcept [](start = 34, length = 8)

Does it compile in Clang? In the past we had issues compiling default constructors with the `noexcept` keyword added like this.